### ISSUE TITLE
Error while building app with graphic package 

### ISSUE THREAD

**Issue by adarsh-technocrat** [entronad/graphic#21]
Hi, @entronad I am getting this error while running the app after installing the graphic package can you please help me with this.

```
/E:/software/flutter/.pub-cache/hosted/pub.dartlang.org/graphic-0.4.1/lib/src/in
teraction/selection/selection.dart:182:36: Error: The getter 'delta' isn't      
defined for the class 'ScaleUpdateDetails'.
 - 'ScaleUpdateDetails' is from 'package:flutter/src/gestures/scale.dart'       
 ('/E:/software/flutter/packages/flutter/lib/src/gestures/scale.dart').
Try correcting the name to the name of an existing getter, or defining a getter 
or field named 'delta'.
              final delta = detail.delta - gesture.preScaleDetail!.delta;       
                                   ^^^^^
/E:/software/flutter/.pub-cache/hosted/pub.dartlang.org/graphic-0.4.1/lib/src/in
teraction/selection/selection.dart:182:68: Error: The getter 'delta' isn't      
defined for the class 'ScaleUpdateDetails'.
 - 'ScaleUpdateDetails' is from 'package:flutter/src/gestures/scale.dart'       
 ('/E:/software/flutter/packages/flutter/lib/src/gestures/scale.dart').
Try correcting the name to the name of an existing getter, or defining a getter 
or field named 'delta'.
              final delta = detail.delta - gesture.preScaleDetail!.delta;
                                                                   ^^^^^        
/E:/software/flutter/.pub-cache/hosted/pub.dartlang.org/graphic-0.4.1/lib/src/ch
art/chart.dart:255:11: Error: No named parameter with the name
'onLongPressCancel'.
          onLongPressCancel: () {
          ^^^^^^^^^^^^^^^^^
/E:/software/flutter/packages/flutter/lib/src/widgets/gesture_detector.dart:216:
3: Context: Found this candidate, but the arguments don't match.
  GestureDetector({
  ^^^^^^^^^^^^^^^
/E:/software/flutter/.pub-cache/hosted/pub.dartlang.org/graphic-0.4.1/lib/src/co
mmon/defaults.dart:162:28: Error: The getter 'delta' isn't defined for the class
'ScaleUpdateDetails'.
 - 'ScaleUpdateDetails' is from 'package:flutter/src/gestures/scale.dart'       
 ('/E:/software/flutter/packages/flutter/lib/src/gestures/scale.dart').
Try correcting the name to the name of an existing getter, or defining a getter
or field named 'delta'.
        (detail) => detail.delta.dx,
                           ^^^^^
/E:/software/flutter/.pub-cache/hosted/pub.dartlang.org/graphic-0.4.1/lib/src/co
mmon/defaults.dart:169:29: Error: The getter 'delta' isn't defined for the class
'ScaleUpdateDetails'.
 - 'ScaleUpdateDetails' is from 'package:flutter/src/gestures/scale.dart'
 ('/E:/software/flutter/packages/flutter/lib/src/gestures/scale.dart').
Try correcting the name to the name of an existing getter, or defining a getter 
or field named 'delta'.
        (detail) => -detail.delta.dy,
```

**Comment by adarsh-technocrat** [comment]
and also this error is found in example apps.

**Comment by adarsh-technocrat** [comment]
**The error is due to getter `'delta' isn't defined for the type 'ScaleUpdateDetails'.`**

**Comment by entronad** [comment]
The [ScaleUpdateDetails](https://api.flutter.dev/flutter/gestures/ScaleUpdateDetails-class.html) is provided by flutter.
In the current stable flutter version 2.5.x, it has the `delta` property: https://github.com/flutter/flutter/blob/18116933e77adc82f80866c928266a5b4f1ed645/packages/flutter/lib/src/gestures/scale.dart#L101
But in the feature version, in and above 2.6.x, they changet the name `delta` to `focalPointDelta`: https://github.com/flutter/flutter/blob/4b330ddbedab445481cc73d50a4695b9154b4e4f/packages/flutter/lib/src/gestures/scale.dart#L101

I guess you used the flutter dev channel, not the stable channel.

I can't upgrade it now because most people use stable channel. I suggest you to switch to stable channel.

**Comment by adarsh-technocrat** [comment]
Thanks, @entronad  !!

**Comment by adarsh-technocrat** [comment]
@entronad  this my flutter doctor

**Comment by adarsh-technocrat** [comment]
> @adarsh-technocrat Er... it seems your flutter version is too old, not too new... I said above that the 2.6.x changed `delta` to `focalPointDelta` before, but I find that in the old version 2.2.x, there is no 'delta' yet: https://github.com/flutter/flutter/blob/f4abaa0735eba4dfd8f33f73363911d63931fe03/packages/flutter/lib/src/gestures/scale.dart#L80
> 
> I suggest you to upgrade your flutter SDK to the "current" version, of which stable version is 2.5.x.
> 
> How unstable the "stable channel" is ...

oooh got it @entronad :)

**Comment by entronad** [comment]
We have constrained flutter version to `'>=2.4.0 <2.6.0'` for `ScaleUpdate.delta` in v0.5.0.
And we will keep upgrading when flutter stable channel gets above 2.6.0.

**Comment by entronad** [comment]
Since the current stable Flutter version is 2.8.0, we published the new Graphic version 0.6.0, which requires Flutter version `'>=2.6.0'`.
If your Flutter version is under 2.6.0, please use Graphic 0.5.1. (We hope you upgrade your Flutter soon and enjoy new Graphic versions.)
